auth-web/src/components/auth/manage-business/NameRequestLookup.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <!-- once in Summary state, need to re-mount to reuse this component -->
+  <div id="business-lookup" v-if="state !== States.SUMMARY">
+    <v-autocomplete
+      :hide-no-data="state != States.NO_RESULTS"
+      :items="searchResults"
+      :loading="state === States.SEARCHING"
+      :name="Math.random()"
+      :search-input.sync="searchField"
+      append-icon="mdi-magnify"
+      autocomplete="chrome-off"
+      autofocus
+      class="mt-5 mb-n2"
+      filled
+      hint="For example: &quot;NR 1234567&quot;"
+      item-text="name"
+      item-value="identifier"
+      label="Name Request Number"
+      no-filter
+      persistent-hint
+      return-object
+      @input="onItemSelected($event)"
+      @keydown.enter.native.prevent
+    >
+      <template v-slot:append>
+        <v-progress-circular
+          v-if="state === States.SEARCHING"
+          color="primary"
+          indeterminate
+          :size="24"
+          :width="2"
+        />
+      </template>
+
+      <template v-slot:no-data>
+        <p class="pl-5 font-weight-bold">
+          No Name Request found
+        </p>
+        <p class="pl-5">
+          Ensure you have entered the correct Name Request.
+        </p>
+      </template>
+
+      <template v-slot:item="{ item }">
+        <v-row class="business-lookup-result pt-1">
+          <v-col cols="2" class="result-identifier">{{item.identifier}}</v-col>
+          <v-col cols="8" class="result-name">{{item.name}}</v-col>
+          <v-col cols="2" class="result-action">
+            <span v-if="item.disabled" class="added">Added</span>
+            <span v-else class="select">Select</span>
+          </v-col>
+        </v-row>
+      </template>
+    </v-autocomplete>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Watch } from 'vue-property-decorator'
+import { BusinessLookupResultIF } from '@/models'
+import BusinessLookupServices from '@/services/business-lookup.services'
+import { Debounce } from 'vue-debounce-decorator'
+import Vue from 'vue'
+import { namespace } from 'vuex-class'
+
+const BusinessModule = namespace('business')
+
+enum States {
+  INITIAL = 'initial',
+  SEARCHING = 'searching',
```

Comment:
Does this need a TYPING state also? And the other changes you made to BusinessLookup?